content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,170 @@
+- Unplug the **programmer** Arduino board from your computer.
+- Disconnect the programmer Arduino board from the target Arduino board.
+
+**Congratulations** you have now successfully burned the bootloader on your Arduino board and can go back to upload sketches like before.
+
+## No SD card
+### Requirements
+- An extra SAMD-based Arduino board to use as the programmer. 
+- A way to make connections to the SWD pins on your target Arduino board.
+
+### Instructions
+- Plug the USB cable of the Arduino board you will be using as a programmer into your computer.
+- Select **Sketch > Include Library > Manage  Libraries…** from the Arduino IDE menu.
+- Wait for the index updates to finish.
+- In the "**Filter your search…**” field type ‘Adafruit DAP library’.
+- Press **Enter**.
+- Click on “**Adafruit DAP library by Adafruit**”.
+- Click the **Install** button.
+- You may now get a dialog asking whether to install the library's dependencies. If so, click the "**Install All**" button.
+- Wait for the installation to finish
+- Close the **Library Manager**
+- Select **File > Example > Adafruit DAP Library > samd21 > flash_MKR_bootloaders** from the Arduino IDE menu. (despite the “MKR” in the sketch name, this also supports the **Nano 33 IoT**).
+- Select your **programmer** Arduino board from the IDE’s **Tools > Board** menu.
+- Select the port of the **programmer** Arduino board from the Arduino IDE’s Tools > Port menu.
+- Select **Sketch > Upload** from the Arduino IDE menu.
+- Unplug the **programmer** Arduino board from your computer.
+- Connect the **programmer** Arduino board to the target Arduino board as follows:
+
+    | Programmer  | Target      |
+    | ----------- | ----------- |
+    | VCC         | +3V3        |
+    | 12          | SWDIO       |
+    | 11          | SWCLK       |
+    | GND         | GND         |
+    | 9           | RESETN      |
+
+    SWD pads on MKR boards other than MKR 1000:
+
+    ![](assets/SWDpadsMKR.png)
+
+    MKR1000 SWD header pinout:
+
+    ![](assets/SWDMKR1000header.png)
+
+    Nano 33 IoT SWD pads:
+
+    ![](assets/SWDpadsNano33IoT.png)
+
+- Plug the USB cable of the **programmer** Arduino board into your computer.
+- Select **Tools > Serial Monitor** from the Arduino IDE’s menu.
+- Select "**No line ending**” from the dropdown menu near the bottom right corner of the **Serial Monitor** window.
+- You should now see some instructions for using the sketch in the **Serial Monitor** output field. 
+
+This includes a menu of the boards supported by the sketch:
+
+   `
+  [...]
+
+   Select Arduino MKR board to erase and flash with bootloader:
+   
+   Z     -> Arduino Zero (6504 bytes)
+   MZ    -> Arduino MKR Zero (6408 bytes)
+   1000  -> Arduino MKR 1000 WIFI (6408 bytes)
+   1010  -> Arduino MKR WIFI 1010 (7984 bytes)
+
+   [...]
+   `
```

Comment:
Something is wrong with the markup here. Check how it is rendered:

https://docscontent-hannes7eicherbootloaderreloadi.gatsbyjs.io/tutorials/generic/bootloader-reloading#instructions-1

![image](https://user-images.githubusercontent.com/8572152/212474078-6e29e8e3-7317-4622-93cb-3cee3809a26d.png)

(note it is not in a code block as intended.

It might be caused by the inconsistent indentation of the text (the first line of the block is indented by two spaces, while the rest is indented three spaces.

I don't know about the Arduino documentation system, but from the standpoint of normal Markdown usage I find this markup very odd. The standard markup for code blocks is a "[fenced code block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks)", where the block content is wrapped with triple backticks (` ``` `):

````markdown
  ```
  [...]

  Select Arduino MKR board to erase and flash with bootloader:

  Z     -> Arduino Zero (6504 bytes)
  MZ    -> Arduino MKR Zero (6408 bytes)
  1000  -> Arduino MKR 1000 WIFI (6408 bytes)
  1010  -> Arduino MKR WIFI 1010 (7984 bytes)

  [...]
  ```
````

Single backticks are normally only used for inline content.

Reply:
You're right and it should indeed be three.